sqlmesh/core/engine_adapter/duckdb.py

```diff
@@ -7,13 +7,14 @@
 from sqlglot import exp
 
 from sqlmesh.core.engine_adapter.base import EngineAdapter
+from sqlmesh.core.engine_adapter.logical_merge import LogicalMerge
 from sqlmesh.core.engine_adapter.shared import DataObject, DataObjectType
 
 if t.TYPE_CHECKING:
     from sqlmesh.core._typing import TableName
 
 
-class DuckDBEngineAdapter(EngineAdapter):
+class DuckDBEngineAdapter(LogicalMerge, EngineAdapter):
```

Comment:
Do you need to subclass both LogicalMerge and EngineAdapter? Is LogicalMerge some sort of mix-in? It looks like LogicalMerge already subclasses EngineAdapter.

Reply:
Originally LogicalMerge was a mixin, but @izeigerman preferred it directly inherit from EngineAdapter, so that's where it stands now. 

My thinking is there will likely be other engine adapter mixins, so I didn't want to remove the EngineAdapter from the DuckDBEngineAdapter inheritance statement and make it seem like LogicalMerge was a new base class.

Open to opinions here.

Reply:
i'd remove it